project/CLI.py

```diff
@@ -19,8 +19,17 @@ def __login(self, args):
             return "Login failed"
         return "Login successful"
 
+    def __create_game(self, args):
+        try:
+            self.gm = self.game.start(args[1])
```

Comment:
Why is this setting the GameMaster (gm) to something related to game?
And why is this calling game.start()?
I think what you wanted here is self.game = Game()?

Reply:
Whoopss, Yeah you're right.

Reply:
When setting  self.game = Game() I am getting an unresolved reference error.  Not sure if that is just because of the way I have things set locally or what.

"This inspection detects names that should resolve but don't. Due to dynamic dispatch and duck typing, this is possible in a limited but useful number of cases. Top-level and class-level items are supported better than instance items."

Reply:
Need to import Game, then it will be Game.Game()

Reply:
Do I need a Try with the game creation? and if so, what kind of error should I be throwing?

Reply:
You need to use the GameFactory instead which i created. But i need help merging my pull request

Reply:
which would look like
self.game = GameFactory().getGame()